runtime/api/Robot.py

```diff
@@ -12,49 +12,494 @@
 >>> from api import Robot
 '''
 import memcache
-
 memcache_port = 12357
 mc = memcache.Client(['127.0.0.1:%d' % memcache_port]) # connect to memcache
 
 motor = {}
 
+def _lookup(name): #Returns actual UID given name
+    return name
+
 def get_motor(name):
     """Returns the current power value for a motor.
 
     :param name: A string that identifies the motor
     :returns: A value between -100 and 100, which is the power level of that motor.
```

Comment:
I think it would make more sense for us to avoid using decimals and keep it to whole numbers since it makes it more intuitive. This range is also native to the grizzlies itself as a way to make the range more understandable. Also, where are you seeing those values because if it doesn't make sense for those ranges to be decimals, I'd like to avoid it if possibles.